opencart 2.2.0.0/upload/catalog/controller/smailyforopencart/cron_customers.php

```diff
@@ -72,6 +72,11 @@ public function index() {
                     die('Error with request to Smaily API, try again later.');
                 }
             }
+            $this->model_smailyforopencart_helper->editSettingValue(
+                'module_smaily_for_opencart_sync',
+                'module_smaily_for_opencart_sync_time',
+                date('Y-m-d H:i:s')
```

Comment:
Please change the date format to include a timezone. This can be easily done by changing the `date` function format string to `c` (more in the [PHP manual](https://www.php.net/manual/en/function.date.php)).

Reply:
Will the format make no difference for the SQL query? just date('c')?